frontend_tests/node_tests/hotkey.js

```diff
@@ -110,7 +110,7 @@ function stubbing(func_name_to_stub, test_function) {
         assert_unmapped(' ');
         assert_unmapped('[]\\.,;');
         assert_unmapped('ABCDEFGHIJKLMNOPQRSTUVWXYZ');
-        assert_unmapped('~!@#$%^*()_+{}:"<>?');
+        assert_unmapped('~!@#$%^()_+{}:"<>?');
```

Comment:
I think you should leave it in this list -- this is for with the compose box open.